assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -100,9 +119,12 @@ const DashboardIdeasWidget = ( {
 	const handleTabUpdate = useCallback(
 		( tabIndex ) => {
 			setActiveTabIndex( tabIndex );
-			setHash( DashboardIdeasWidget.tabIDsByIndex[ tabIndex ] );
+			setQueryString( {
+				'idea-hub-tab': DashboardIdeasWidget.tabIDsByIndex[ tabIndex ],
+				page: queryParamPage,
+			} );
```

Comment:
Can we not update only the one query param without writing the whole query string? Looks like we even have our own custom hook for this from a while back: https://github.com/google/site-kit-wp/blob/develop/assets/js/hooks/useQueryArg.js

Reply:
Hah, nice, I totally forgot about it. Let's use that then! 👍🏻